src/amo/pages/Home/index.js

```diff
@@ -146,7 +147,9 @@ export class HomeBase extends React.Component {
         {curatedMozillaCollections.map(({ collectionSlug, title }) => (
           <li className="Home-SubjectShelf-list-item" key={collectionSlug}>
             <Link
-              to={`/collections/mozilla/${collectionSlug}/`}
+              to={`/collections/${_config.get(
```

Comment:
If you set a `const` as I suggested above, that would simplify this code as well, and you wouldn't even need for `renderCuratedCollections` to access `_config` at all.

Reply:
:+1: